src/sentry/features/manager.py

```diff
@@ -327,10 +327,13 @@ def batch_has(
         Will only accept one type of feature, either all ProjectFeatures or all
         OrganizationFeatures.
         """
+        sample_rate = 0.01
```

Comment:
Any reason to define this separately here instead of directly setting it on the metrics.timer?

Reply:
I was mainly following the pattern `has` was using for this, but no strong reason to do it this way.